src/pudl/metadata/resources/eia930.py

```diff
@@ -192,6 +192,31 @@
         "etl_group": "eia930",
         "create_database_schema": False,
     },
+    "out_eia930__hourly_demand": {
+        "description": (
+            """EXPERIMENTAL / WORK-IN-PROGRESS, 2025-04-04.
+
+This aggregates imputed demand from the ``out_eia930__hourly_operations`` table.
+It aggregates demand to the level of EIA regions, interconnects, and the contiguous United States."""
+        ),
+        "schema": {
+            "fields": [
+                "datetime_utc",
+                "code",
+                "aggregation_group",
```

Comment:
I think this field was used for `gridpathtoolkit`, and it seemed like it made sense here. Need to override the field metadata if we want to use this column name though.

Reply:
We have infrastructure for resource (table) specific and data source specific column definitions, down at the bottom of `fields.py` which will override the generic field type/descriptions/constraints.  So if this will show up with different descriptions in the two diferent data sources, probably we should define the fields for the data sources, rather than globally.

However, I had trouble making this work in the SEC 10-K PR.  But it's supposed to work! And it does work for some other data sources, so I think there was just a bit of plumbing that I was missing.

Reply:
What values do we expect to find in `aggregation_group`? `conus`, `eastern`, `western`, `ercot`, region codes? individual BA codes? Should explain that in the table description.

Reply:
Maybe I'm not understanding the different roles of `aggregation_group` and `code` in this context.

Reply:
In the GridPath RA Toolkit table, I think `aggregation_group` corresponds to the meaning that `code` has here -- it's the identifier of the individual group, not a category that describes what kind of group it is. Maybe it would make sense here to have

* `aggregation_group` -- the ID of the group, like `ERCOT` or `CONUS`
* `aggregation_level` -- the granularity of aggregation, e.g. `conus`, `interconnect`, `region_eia`

Reply:
I think the `aggregation_group/level` combo makes sense. Fixed